pkg/oam/util/helper_test.go

```diff
@@ -292,3 +292,41 @@ var _ = Describe("Test unstructured related helper utils", func() {
 		}
 	})
 })
+
+var _ = Describe("Test GenTraitName helper utils", func() {
+	It("Test generate trait name", func() {
```

Comment:
This test actually relies on the ComputeHash function. I wonder if we can have an "expected" value that is not computed by the ComputeHash function?

Reply:
done